react/features/dropbox/functions.web.js

```diff
@@ -62,7 +52,7 @@ export function _authorizeDropbox(
             return {
                 token: resp.result.access_token,
                 rToken: resp.result.refresh_token,
-                expireDate: getTokenExpiresAtDate(resp.result.expires_in)
+                expireDate: resp.result.expires_in * 1000
```

Comment:
this is the number of seconds until token expires , <=> 4 hrs, not the timestamp in seconds

Reply:
new Date(Date.now() + ( resp.result.expires_in* 1000)).getTime()